producers/smd_producer.py

```diff
@@ -508,14 +588,14 @@ def get_sd_file(write_dir, exp, hutch):
     except:
         userDataCfg[det.name] = det.params_as_dict()
 
-#is EOODet exists, save this later.
+# is EOODet exists, save this later.
 if EODet is None:
     Config={'UserDataCfg':userDataCfg}
     small_data.save(Config)
 
 end_setup_dets = time.time()
 
-if args.tiff:
+if args.tiff: # this needs to be done for S3DF
```

Comment:
Yes, I think we use the data directory and replace /xtc with /scratch & keep the rest?

Reply:
I can add that in my pull request (and latest when MEC wonders where their tiff files are during their first user experiment in S3DF....)